src/Pages/CardEditor.js

```diff
@@ -59,17 +79,25 @@ export const CardEditor = () => {
 			633
 		)
 
-		pdf.save('canvas.pdf')
-		return pdf.output('blob')
-	}
+		pdf = addWaterMark(pdf);
+		// pdf.save('canvas.pdf')
+		//return pdf.output('blob')
 
-	const postPdfAndCheckout = async () => {
-		const pdf = await exportToPDF()
-		const url = await postToCheckout('price_1MLxogFPiM3jeCEiCsDS0lDy', pdf)
-		setredirectUrl(url)
-		console.log(url)
+		const url = await postToCloudinary(pdf.output('blob'))
+
+		setPDFUrl(url)
+		console.log(url);
+
+		navigate('checkout')
 	}
 
+	// const postPdfAndCheckout = async () => {
```

Comment:
Let's not repeat logic in other files

Reply:
I commented this just in case I needed to go back comments are gone now